integration/util_test.go

```diff
@@ -84,6 +85,7 @@ const (
 	ubuntuImage   = "ubuntu:23.04"
 	drupalImage   = "drupal:10.0.2"
 	rabbitmqImage = "rabbitmq:3.11.7"
+	zotImage      = "ghcr.io/project-zot/zot-linux-" + runtime.GOARCH + ":v2.0.0-rc6"
```

Comment:
nit: Can we use string format notation:  `fmt.Sprintf`

Also to keep some consistency, can we rename it to something like `oci11RegistryImage` and move it up to where the `oci10RegistryImage` is initialized?

Reply:
Sure, I can rename the variable. I can't use string format notation as it's a constant so it'll unfortunately have to look a tad ugly, unless we can find some workaround.

Reply:
Ah, right. I think it's fine then.